libdnf/comps/environment/query.cpp

```diff
@@ -43,10 +44,15 @@ namespace libdnf::comps {
 EnvironmentQuery::EnvironmentQuery(const BaseWeakPtr & base) : base(base) {
     libdnf::solv::Pool & pool = get_pool(base);
 
-    std::map<std::string, std::vector<Id>> available_map;
+    // Map of available environments:
+    //     For each environmentid (SOLVABLE_NAME) have a vector of (repoid, solvable_id) pairs.
+    //     Each pair consists of one solvable_id that represents one definition of the environment
+    //     and repoid of its originating repository (in the opposite order).
```

Comment:
Nitpicking, but _"in **reverse** order"_, though the sorting is done explicitly after the map is filled up so I'd leave out the note here.

Reply:
In this case, I just meant the order within the pair, because I first had to mention solvable_id and then repoid, but the pair is (repoid, solvable_id). But I see it's confusing, so I'll remove it. And it's mentioned before, so it will be fine without it.